app/views/proposals/query.html.haml

```diff
@@ -0,0 +1,6 @@
+.inset
+  %h1
+    Proposals participated in
+    = datespan_header
+
+  = client_partial (current_user.client_slug || 'ncr'), "proposal_list", locals: {proposals: @proposals}
```

Comment:
Going to revert the change which set the default as it causes lots of errors during tests. Notably, listing pages use the NCR listing, which includes fields not present on a non-ncr model.